bin/cron/applab_datasets/spotify

```diff
@@ -15,7 +15,7 @@ VIRAL_50_WORLD_URL = 'https://spotifycharts.com/viral/global/daily/latest/downlo
 
 def get_top200_data(url)
   records = {}
-  columns = ['Position', 'Track Name', 'Artist', 'Streams', 'URL']
+  columns = ['id', 'Track Name', 'Artist', 'Position', 'Streams', 'URL']
```

Comment:
Nit: Not sure whether this matters, but I thought the lower-case `id` looked a little funny next to all the other capitalized column names.

Reply:
i.e. could be ID instead.

Reply:
I think it's probably better to keep it consistent with every other data table (always lowercase `id`)
If we want to change this across the board for the data tab, that's definitely doable, though.